README.md

```diff
@@ -157,7 +316,7 @@ BuildmasterConfig = {
 ```
 
 
-## Docker build tool
+## Define docker images
```

Comment:
I don't see that ursabot contains Docker images or even Dockerfiles--am I missing something?